From 3921b7815ae7ac7c7ddd9e4e6fced4fdf3cccbd1 Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Wed, 29 Jun 2011 12:05:52 +0200
Subject: [PATCH] Issue #1203684: fix use of undefined constant FILTER_FORMAT_DEFAULT
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

When using profiler with Drupal 7 we get this message:

  Use of undefined constant FILTER_FORMAT_DEFAULT -
  assumed 'FILTER_FORMAT_DEFAULT' in profiler_install_nodes()
  (line 327 of .../libraries/profiler/profiler_api.inc).

Use filter_fallback_format() instead of FILTER_FORMAT_DEFAULT.
---
 profiler_api.inc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/profiler_api.inc b/profiler_api.inc
index c355fe7..4e4110a 100644
--- a/profiler_api.inc
+++ b/profiler_api.inc
@@ -324,7 +324,7 @@ function profiler_install_nodes($nodes, $config, &$identifiers) {
       'teaser' => '',
       'log' => '',
       'created' => '',
-      'format' => FILTER_FORMAT_DEFAULT,
+      'format' => filter_fallback_format(),
       'uid' => !empty($account) ? $account->uid : 0,
     );
     $node = (object) array_merge($default, $properties);
-- 
1.7.5.4

